packages/adminrouter/extra/src/README.md

```diff
@@ -197,56 +216,57 @@ http {
 ```
 
 ### Enforcing code reuse
-The EE repository contains only EE directories, all common/agent-common/master-common
-code resides in Open repository. This way EE repository becomes an overlay on
-top of Open. Only `nginx.(master|agent).conf` are overwritten while applying EE
-repository on top of Open during DC/OS image build. EE DC/OS image build scripts
-remove all open directories from the Open repository before applying EE repository on
-top of it.
+The EE repository contains only EE directories, all
+common/agent-common/master-common code resides in Open repository. This way EE
+repository becomes an overlay on top of Open. Only `nginx.(master|agent).conf`
+are overwritten while applying EE repository on top of Open during DC/OS image
+build. EE DC/OS image build scripts remove all open directories from the Open
+repository before applying EE repository on top of it.
 
 This is not a bulletproof solution for preventing code
 duplication (developers can simply start putting copies of code to both
 `open/` and `ee/` directories) but it makes it easier to reuse the
 code and encourages good behaviours.
 
 ### Lua code deduplication
-It is not strictly required to provide the same level of flexibility for Lua code
-as it is the case for NGINX includes and thus it's possible to simplify the code
-a bit. It is sufficient to just differentiate Lua code basing on the repository flavour.
-Both agents and masters can share the same Lua code.
+It is not strictly required to provide the same level of flexibility for Lua
```

Comment:
> It is not strictly required

Can we remove "strictly"?

Otherwise your readers think you are a policeman with a stick, ready to punch. Strictly.